R/recall_data.R

```diff
@@ -191,28 +200,41 @@ recall_details <- function(recall_number, limit = 25) {
             call. = FALSE)
     }
 
-    compiled_df <- vector(mode = "list", length = length(recall_number))
+
     i <- 1
     for (single_number in recall_number) {
 
+        #initialize dataframe
+        if (i == 1){
+
```

Comment:
I think this block is the only thing that should be updated, everything else looks good

Reply:
plan is to deal with this later